cache/lru_cache.cc

```diff
@@ -560,7 +560,7 @@ LRUHandle* LRUCacheShard::Lookup(const Slice& key, uint32_t hash,
                                  is_in_sec_cache);
     if (secondary_handle != nullptr) {
       e = reinterpret_cast<LRUHandle*>(
```

Comment:
`static_cast` can be used for better hygiene (cleaner looking; detect possible bugs under future modification)